src/conversion/docs/readme.md

```diff
@@ -0,0 +1,11 @@
+This conversion filter is dedicated to convert from/to different units.
```

Comment:
I think we will need also speed, but not sure yet
https://gist.github.com/letanure/4a81adfbda16b52d25ed